src/volcartesian/volcartesian.cpp

```diff
@@ -657,6 +682,31 @@ double VolCartesian::evalCellSize(long id) const
 {
 	BITPIT_UNUSED(id);
```

Comment:
This is an override, however there is also a evalCellSize function that has no arguments. That function is now private, but I can make it public.

Reply:
I was referring to the macro BITPIT_UNUSED(). If I understand correctly, it is there just to avoid the "unused variable" compiler warning. To get rid of the warning (without using any custom macro), you can just drop "id" from the list of input. In this way variable "id" is never declered (and used) within the method implementation, thus avoiding the warning.
(This is just to avoid as much as possible macros)

Reply:
Ah, ok. I'm using BITPIT_UNUSED just out of habit. I we all agree, we can stop using that macro and just drop the unused arguments throughout all bitpit (let just check if Doxygen is happy with such a change).